mods/d2k/chrome/lobby-players.yaml

```diff
@@ -181,13 +181,13 @@ Container@LOBBY_PLAYER_BIN:
 							Width: 165
 							Height: 25
 							Text: Name
-						Button@KICK:
-							X: 155
-							Y: 2
-							Width: 25
+						DropDownButton@PLAYER_ACTION:
+							X: 15
+							Width: 165
 							Height: 23
```

Comment:
This height was wrong, but I figured it was easier to fix this in my auth PR which is already touching this code than to go through a fixup cycle here.